src/Microsoft.Health.Fhir.SqlServer/Resources.resx

```diff
@@ -126,4 +126,10 @@
   <data name="CyclicIncludeIterateNotSupported" xml:space="preserve">
     <value>Cyclic include iterate queries are not supported.</value>
   </data>
-</root>
\ No newline at end of file
+  <data name="SchemaVersionNeedsUpgrading" xml:space="preserve">
```

Comment:
nit: probably SchemaVersionNeedsUpgrading -> SchemaVersionNeedsUpgrade

Reply:
Reworded to "SchemaVersionNeedsToBeUpgraded" in 229f90c.